cmd/kk/pkg/config/templates/cluster.go

```diff
@@ -64,6 +64,9 @@ spec:
     ## multus support. https://github.com/k8snetworkplumbingwg/multus-cni
     multusCNI:
       enabled: false
+  storage:
+    openebs:
+      basePath: /var/openebs
```

Comment:
I think we just only show this feature in the [config-example](https://github.com/kubesphere/kubekey/blob/master/docs/config-example.md) because that will be confusing to users who do not deploy the `openebs`.